contracts/interfaces/IERC20Metadata.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.10;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+interface IERC20Metadata is IERC20 {
```

Comment:
Is this interface required?

Reply:
Not really, I did add it to the `IWrappedFCashComplete` interface. But we don't access these additional methods anywhere, so I removed the interface again.